jupyter-org-client.el

```diff
@@ -845,6 +845,36 @@ the width or height of the image."
         metadata
       (jupyter-org-image-link file width height))))
 
+(defun jupyter-pandoc-convert (from to from-string &optional callback)
```

Comment:
Could you move this to `jupyter-mime.el` since it could be useful in other contexts in the future. You can place it at the end of the file with a new comment heading:

```emacs-lisp
;;; Pandoc
```